content/docs/1.1.1/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -70,7 +137,7 @@ This page covers the following topics:
    
    For Google Cloud Storage, you can find the region codes [here.](https://cloud.google.com/storage/docs/locations)
 
-5.  In the Backup section set **Backup Target Credential Secret** to:
+4.  In the Backup section set **Backup Target Credential Secret** to:
 
     ```
     aws-secret
```

Comment:
Since you changed the ordering the line below the ``` aws-secret ``` needs to be updated.

Reply:
I've updated it, please check.